tests/steps/steps.py

```diff
@@ -404,20 +468,18 @@ def send_msig_txn(context):
 
 @then("the transaction should go through")
 def check_txn(context):
-    last_round = context.acl.status()["lastRound"]
+    wait_for_algod_transaction_processing_to_complete()
     assert "type" in context.acl.pending_transaction_info(
         context.txn.get_txid()
     )
-    context.acl.status_after_block(last_round + 2)
     assert "type" in context.acl.transaction_info(
         context.txn.sender, context.txn.get_txid()
     )
-    assert "type" in context.acl.transaction_by_id(context.txn.get_txid())
```

Comment:
@algochoi Can you restore the deleted assertion?

I'm _mostly_ confident it works now that the waiting approach changed.  See corresponding change in https://github.com/algorand/java-algorand-sdk/pull/350/commits/fbcbf63efaa65f31d3cf011109b070b283f296db#diff-2c7314dd90c34b904f243500da609439f53ef9791207a076eec677cb0e8d2611L829-L830.

Reply:
Good point - done!